README.md

```diff
@@ -1,51 +1,238 @@
+
+![Text generation using LLaMa 3.2 model running on Intel ARC770 dGPU](./samples/generation.gif)
+
+## Supported Generative AI scenarios
+
+OpenVINO™ GenAI library provides very lightweight C++ and Python APIs to run following Generative Scenarios:
+ - Text generation using Large Language Models. For example, chat with local LLaMa model
+ - Image generation using Diffuser models, for example generation using Stable Diffusion models
+ - Speech recognition using Whisper family models
+ - Text generation using Large Visual Models, for instance Image analysis using LLaVa or miniCPM models family
+
+Library efficiently supports LoRA adapters for Text and Image generation scenarios:
+- Load multiple adapters per model
+- Select active adapters for every generation
+- Mix multiple adapters with coefficients via alpha blending
+
+All scenarios are run on top of OpenVINO Runtime that supports inference on CPU, GPU and NPU. See [here](https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html) for platform support matrix.
+
+## Supported Generative AI optimization methods
+
+OpenVINO™ GenAI library provides transparent way to use state of the art generation optimizations:
+- Speculative decoding that employs two models of different size and uses large model to periodically correct results of small model. See [here](https://pytorch.org/blog/hitchhikers-guide-speculative-decoding/) for more detailed overview
+- Lookahead decoding that attempts to guess multiple tokens based on historical context and corrects this guess using LLMs. See [here](https://lmsys.org/blog/2023-11-21-lookahead-decoding/) for more detailed overview
+- KVCache token eviction algorithm that reduces size of the KVCache by pruning less impacting tokens.
+
+Additionally, OpenVINO™ GenAI library implements continuous batching approach to use OpenVINO within LLM serving. Continuous batching library could be used in LLM serving frameworks and supports following features:
+- Prefix caching that caches fragments of previous generation requests and corresponding KVCache entries internally and uses them in case of repeated query. See [here](https://google.com) for more detailed overview
+
+## Installing OpenVINO GenAI
+
+```sh
+    # Installing OpenVINO GenAI via pip
+    pip install openvino-genai
+
+    # Install optimum-intel to be able to download, convert and optimize LLMs from Hugging Face
+    # Optimum is not required to run models, only to convert and compress
+    pip install optimum[openvino]
+
+    # (Optional) Install (TBD) to be able to download models from Model Scope
+    #pip install optimum[openvino]
+```
+
+## Performing text generation 
+<details>
+For more examples check out our [LLM Cheat Sheet](https://docs.openvino.ai)
```

Comment:
should it point to https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide.html ?

Reply:
Yes, I will use this now, but original intent was to make cheat sheet for GenAI specifically. 
We can update readme later.